reportportal/templates/analyzer-statefulset.yaml

```
@@ -62,6 +62,10 @@ spec:
         - name: UWSGI_WORKERS
           value: "{{ .Values.serviceanalyzer.uwsgiWorkers }}"
         {{- end }}
+        {{- range .Values.serviceanalyzer.env }}
+        - name: {{ .name }}
+          value: {{ .value }}
+        {{- end }}
```

```suggestion
        {{- range .Values.serviceanalyzer.env }}
        {{- range $key, $value := .Values.serviceanalyzer.env }}
        - name: {{ $key }}
          value: {{ $value | quote }}
        {{- end }}
```
In this case `env` parameter will be just simple:
```yaml
env:
  ES_CHUNK_NUMBER: 2000
```